python/dgl/nn/pytorch/explain/pgexplainer.py

```diff
@@ -196,6 +195,10 @@ def train_step(self, graph, feat, tmp):
             number of nodes, and :math:`D` is the feature size.
         tmp : float
             The temperature parameter fed to the sampling procedure.
+        kwargs : dict
+            Additional arguments passed to the GNN model. Tensors whose
+            first dimension is the number of nodes or edges will be
+            assumed to be node/edge features.
```

Comment:
"Tensors whose first dimension is the number of nodes or edges will be assumed to be node/edge features." 

Do we need this sentence?

Reply:
Not really, fixed.